### ISSUE TITLE
Report Accessibility: Audit details have duplicate ids

### ISSUE THREAD

**Issue by johnemau** [GoogleChrome/lighthouse#9432]
#### Provide the steps to reproduce
1. Run LH on https://example.com
2. Inspect the html report (text editor or browser DevTools)
3. Search for the text id="document-title" (or just "document-id" in the DevTools).

#### What is the current behavior?
2 search results

![image](https://user-images.githubusercontent.com/309310/61668233-f5c35c80-ac90-11e9-993f-163b12f1f230.png)

#### What is the expected behavior?
One or fewer results

#### Environment Information
* Affected Channels: CLI, Node, Extension, DevTools
* Lighthouse version: 5.1.0
* Node.js version: v12.6.0
* Operating System: Win10

#### Notes
* This is failure of [WCAG 4.1.1 Parsing - Level A](https://www.w3.org/WAI/WCAG21/quickref/#parsing).
* Root cause: https://github.com/GoogleChrome/lighthouse/pull/5058#discussion_r184791923.
* This impacts https://github.com/GoogleChrome/lighthouse/pull/9421

**Comment by paulirish** [comment]
I know duplicated IDs does affect forms/labels, etc.. but I don't think these particular duplicate IDs will have a real effect. 

(This argument is partly motivated by the fixes to this being rather awkward and unattractive. ;)

We'd like to wontfix this one. @johnemau 	wdyt?

**Comment by johnemau** [comment]
OK with me to wontfix.

Because the IDs are not used in any ARIA attributes or as you mentioned labels this won't cause any issues for assistive technology like screen readers, which addresses the [intent of WCAG 4.1.1](https://www.w3.org/TR/WCAG20-TECHS/F77.html) 👍
